lib/logger.js

```diff
@@ -11,8 +11,44 @@ Hoek.assert(process.env.LOG_LEVEL, 'Must define LOG_LEVEL');
 var stream = new PrettyStream();
 stream.pipe(process.stdout);
 
+function requestSerializer(req) {
+  return {
+    path: req.path,
+    method: req.method,
+    headers: req.headers
+  };
+}
+
+function responseSerializer(res) {
+  return {
+    payload: res.output.payload
+  };
+}
+
+function errorSerializer(res) {
+  if (!res.data) {
```

Comment:
The problem is that you're using bunyan for "abstract logging" (when usually it is used for logging different kinds of data and then selecting which set to use) which sorta negates the purpose of the serializer altogether. In fact, right now, you are doing two sets of serialization. I just think it makes more sense to do:
```js
{
  error: res.data.error
}
```
versus
```js
{
  error: res.data
}
```

But this is not something that is critical